goreleaser_files/.goreleaser-docker-build-api.yml

```diff
@@ -14,6 +14,10 @@ env:
   - DOCKER_BUILDX_CACHE_TO={{ if index .Env "DOCKER_BUILDX_CACHE_TO"  }}{{ .Env.DOCKER_BUILDX_CACHE_TO }}{{ else }}type=inline{{ end }}
   # Build image with commit sha tag
   - IMAGE_TAG_SHA={{ if index .Env "IMAGE_TAG_SHA"  }}{{ .Env.IMAGE_TAG_SHA }}{{ else }}{{ end }}
+  # Build Sandbox Image
+  - SANDBOX_IMAGE={{ if index .Env "SANDBOX_IMAGE"  }}{{ .Env.SANDBOX_IMAGE }}{{ else }}{{ end }}
```

Comment:
Is this `if/else` necessary here?

Reply:
Ok, I guess that is the how templating is configured for GoReleaser. We could maybe improve this with `envOrDefault`: https://goreleaser.com/customization/templates/#functions

However, this has nothing to do with this PR. Maybe in another one.